azure-pipelines.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   pool:
     vmImage: 'windows-2019'
   variables:
-    CMAKE_EXTRA_FLAGS: '-DENABLE_WARNINGS_AS_ERRORS=Off -DUMPIRE_ENABLE_FILESYSTEM=On -DENABLE_TOOLS=Off -DENABLE_BENCHMARKS=Off -DBLT_CXX_STD="" -DCMAKE_CXX_STANDARD=17'
+    CMAKE_EXTRA_FLAGS: '-DENABLE_WARNINGS_AS_ERRORS=Off -DUMPIRE_ENABLE_FILESYSTEM=On -DENABLE_TOOLS=Off -DENABLE_DEVELOPER_BENCHMARKS=On -DENABLE_BENCHMARKS=Off -DBLT_CXX_STD="" -DCMAKE_CXX_STANDARD=17'
```

Comment:
I think I remember this being discussed, but cannot remember the outcome.  Why are benchmarks being run on Azure?  I think that reliable benchmarks should be from resources where we have more control of what the physical resources are.

Reply:
From what I understand, we just wanted to compile the benchmarks to make sure they can still be built. The google benchmarks are disabled, so really only the stress tests will be compiled.

Reply:
@kab163 is right. Also, remember that the CMake flags control what is built, not what is run.